package.json

```
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "tsx ./scripts/build.ts",
     "dev": "tsx ./scripts/dev.ts",
-    "lint": "concurrently \"pnpm:lint:*\"",
+    "lint": "concurrently \"pnpm:lint:*(!fix)\"",
```

I would suggest this renaming and update of the scripts to also take advantage of the cache:

```suggestion
    "lint": "eslint . --cache --cache-location 'node_modules/.cache/eslintcache' --max-warnings 0",
    "lint:fix": "eslint . --fix --cache --cache-location 'node_modules/.cache/eslintcache' --max-warnings 0",
     "format": "prettier . --check --cache --cache-location='node_modules/.cache/prettiercache'",
     "format:fix": "prettier . --write --cache --cache-location='node_modules/.cache/prettiercache' --log-level=warn",
     "typecheck": "tsc --noEmit"
```

---

Rename: Sure.

Oh nice. TIL they all added caches.
Let me check if CI caches node_modules or pnpm store...